Python/bltinmodule.c

```diff
@@ -1449,6 +1470,43 @@ map_next(mapobject *lz)
         PyMem_Free(stack);
     }
     return result;
+check:
+    if (PyErr_Occurred()) {
+        if (!PyErr_ExceptionMatches(PyExc_StopIteration)) {
+            // next() on argument i raised an exception (not StopIteration)
```

Comment:
This comment explains literally what the code reads. I don't think it adds much value. Ditto for L1501.

Reply:
I think this was just C/C from `zip` which  included this kind of comment.

Reply:
Yeah, I was still waiting for a response on what do with it in that case.